### PR TITLE
[iOS] Add Privacy Manifest to PayPal Module

### PR DESCRIPTION
**Pull request by agedd**
### Summary of changes

- Adds `PrivacyInfo.xcprivacy` to the`BraintreePayPal` module
- Adds the corresponding changes to `Package.swift` and `Braintree.podspec` files

### Checklist

- ~Added a changelog entry~

- @agedd

### CODE REVIEW

**Review by scannillo — APPROVED** [review]

File: Braintree.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1747,7 +1754,6 @@
 				BEF177A52B51C7ED004F0F35 /* SEPADebitAccountsRequest.swift */,
 				BEF177A72B51D6A7004F0F35 /* SEPADebitRequest.swift */,
 				BE0B163E27E0CD7C00868C77 /* SEPADirectDebitAPI.swift */,
-				62DE8FBE2B9656BF00F08F53 /* PrivacyInfo.xcprivacy */,
```
Comment (jaxdesmarais) [inline Braintree.xcodeproj/project.pbxproj:1750]: Is this diff expected? It looks to be removing `PrivacyInfo.xcprivacy` from the SEPA module.

File: Braintree.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -2620,7 +2629,6 @@
 			isa = PBXResourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				62DE8FBF2B9656BF00F08F53 /* PrivacyInfo.xcprivacy in Resources */,
```
Comment (jaxdesmarais) [inline Braintree.xcodeproj/project.pbxproj:2623]: Same question here just want to make sure this removal is expected

File: Braintree.xcodeproj/project.pbxproj (hunk 1 above)
Comment (agedd) [reply Braintree.xcodeproj/project.pbxproj]: ahhh thanks for letting me know! yea it's def a bad merge 😅 - fixed in ba5b597fa1ece265495740a52081676163abbb5d

File: Braintree.xcodeproj/project.pbxproj (hunk 2 above)
Comment (agedd) [reply Braintree.xcodeproj/project.pbxproj]: fixed in ba5b597fa1ece265495740a52081676163abbb5d

**Review by jaxdesmarais — APPROVED** [review]

**Pull request merged**